GameServerLib/Logic/Chatbox/Commands/MobsCommand.cs

```diff
@@ -1,16 +1,16 @@
 ﻿using ENet;
 using LeagueSandbox.GameServer.Core.Logic;
 using LeagueSandbox.GameServer.Core.Logic.PacketHandlers;
-using LeagueSandbox.GameServer.Core.Logic.PacketHandlers.Packets;
 using LeagueSandbox.GameServer.Logic.GameObjects;
 using LeagueSandbox.GameServer.Logic.Packets;
 using LeagueSandbox.GameServer.Logic.Players;
 using System.Linq;
+using LeagueSandbox.GameServer.Logic.Packets.PacketHandlers.Handlers;
```

Comment:
Is this sub-namespace necessary at all? How about just `Packets.PacketHandlers`? Double handlers namespace seems a bit redundant to me.